kano_world/functions.py

```diff
@@ -122,7 +121,11 @@ def is_registered():
 
 
 def has_token():
-    return 'token' in load_profile()
+    kw = mercury.KanoWorld(kw_url)
+    if not kw.load_data():
+        return None
+
+    return kw.get_token()
```

Comment:
Looks like this function used to simply return whether or not a token exists, not what it actually is so methinks that we only care about this and this function should simply return a boolean.

Also, we might want to put the `load_data()` into the constructor of the `KanoWorld()` class so we don't need to do it here.

Reply:
You're absolutely right - fixed and tested.